docs/spec.yml

```diff
@@ -0,0 +1,157 @@
+openapi: 3.0.0
+info:
+  version: 1.0.0
+  title: Document Retrieval API
+paths:
+  /:
+    get:
+      summary: Debug Route
+      responses:
+        '200':
+          description: OK
+
+  /version:
+    get:
+      summary: Get API Version
+      responses:
+        '200':
+          description: OK
+
+  /document-retrieval:
+    get:
+      summary: Retrieve a Document
+      parameters:
+        - name: vinNumber
+          in: query
+          description: VIN number
+          required: false
+          schema:
+            type: string
```

Comment:
Should there also be some sort of reference here to the required regex/pattern as per docs?

https://swagger.io/docs/specification/data-models/data-types/#pattern